docs/data-sources/substack.mdx

```diff
@@ -0,0 +1,16 @@
+---
+title: "📝 Substack"
+---
+
+To add any Substack data sources to your app, just add the sitemap.xml of that url as the source and set the data_type to `substack`.
+
+```python
+from embedchain import Pipeline as App
+
+app = App()
+
+# source: for any substack just add the sitemap.xml url
+app.add('https://www.lennysnewsletter.com/sitemap.xml', data_type='substack')
```

Comment:
Do the user have to enter the full url with `sitemap.xml` ? I think we should just assume that sitemap.xml is present and then try to fetch it. If we fail, then we should return error to the user

Reply:
I am going to merge it for now but lets do this change in a follow up PR.